distributor.sample.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<composer-distributor xmlns="https://phar.io/composer-distributor"
+                      packageName="phar-io/phive"
+                      keyDirectory="keys">
+
```

Comment:
We should probably add a schema location reference to this document:

```
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:schemaLocation="https://phar.io/composer-distributor.xsd"`
```

Or something?

We should probably consider specifying a version identifier somehow...

Reply:
what do you think about

    https://phar.io/data/composer-distributor/1.0/distributor.xsd

Reply:
So far, phar.io does not have any published schema urls, so we are relatively free to choose.

We also are sadly quite inconsistent already.

In terms of namespaces, a quick `ack` over all phar.io projects on my disc shows two patterns.

Phive:

- https://phar.io/auth
- https://phar.io/phive-auth
- https://phar.io/phive/installdb
- https://phar.io/phive
- https://phar.io/repository-list
- https://phar.io/repository

And our manifest:

- https://phar.io/xml/manifest/1.0


Sadly, I today think only the manifest format makes sense. :-(

In terms of schema location, it probably makes sense to derive the that from the namespace.
So adopting your proposal:

`https://phar.io/xml/distributor/1.0/schema.xsd`

And I guess we just have to live with the other pattern in `phive` as changing that would be a major BC break with nothing to win :/

Reply:
`https://phar.io/xml/distributor/1.0/schema.xsd` it is